judge/models/tests/test_contest.py

```diff
@@ -210,6 +275,197 @@ def test_hidden_scoreboard_contest_methods(self):
         }
         self._test_object_methods_with_users(self.hidden_scoreboard_contest, data)
 
+    def test_contest_hidden_scoreboard_contest_methods(self):
+        data = {
+            'staff_contest_edit_own': {
```

Comment:
Hmm... do we need to test these specific staff users for all the types of hidden scoreboards? There doesn't seem to be anything special in the methods that differentiates these users. At the very least, I think we should remove the other methods besides `can_see_own_scoreboard`, `can_see_full_scoreboard`, and `has_completed_contest` from all three tests. They don't really add anything (Maybe it's better to test everything together, but personally I try to make each test function to focus on a specific section of the contest code and then one function `test_basic_contest_methods` to test everything together).

Reply:
We don't need the specific staff users for these test, but I thought it would be nice to keep it as a redundancy.